guides/moving-to-v2.md

```diff
@@ -57,19 +58,21 @@ coder workspaces list
 coder2 list
 ```
 
-
 ### Users
 
-If you 
+Like Coder v1, you can [enable SSO via OpenID Connect](https://coder.com/docs/coder-oss/latest/install/auth#step-2-configure-coder-with-the-openid-connect-credentials) so that any user in your federation can log in. Coder v2 optionally supports [GitHub (Enterprise)](https://coder.com/docs/coder-oss/latest/install/auth#step-1-configure-the-oauth-application-in-github) and [username/password](https://coder.com/docs/coder-oss/latest/users) authentication.
```

Comment:
Coder v1 also supports built-in authentication so that should  be mentioned.

Reply:
done via a table